storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -312,6 +312,13 @@
     // the topology.
     public static final String TOPOLOGY_SCHEDULER_STRATEGY = "topology.scheduler.strategy";
 
+    /**
+     * When DefaultResourceAwareStrategy or GenericResourceAwareStrategy is used,
+     * it will arrange unassigned executors based a particular order.
+     * If this config is set to true, the arrangement will be made by network proximity needs.
+     */
+    public static final String TOPOLOGY_RAS_SCHEDULE_EXECUTORS_BY_PROXIMITY_NEEDS = "topology.ras.schedule.executors.by.proximity.needs";
```

Comment:
Maybe `topology.ras.order.executors.by.proximity.needs` has more accurate meaning. This only changes how the unassigned executors are sorted. How they are scheduled still depend on some other things too

Reply:
Yes, fixed